build.psm1

```diff
@@ -188,6 +190,12 @@ function Start-PSBuild {
     # verify we have all tools in place to do the build
     $precheck = precheck 'dotnet' "Build dependency 'dotnet' not found in PATH. Run Start-PSBootstrap. Also see: https://dotnet.github.io/getting-started/"
 
+    $dotnetIntalledVersion = (dotnet --version)
+    If ( $dotnetIntalledVersion -ne $dotnetRequiredVersion ) {
+        Write-Warning ".Net Core version mismatch: installed version is $dotnetIntalledVersion, required version is $dotnetRequiredVersion.`nRun Start-PSBootstrap and Start-PSBuild -Clean -Restore.`nAlso see: https://dotnet.github.io/getting-started/"
```

Comment:
`Install-Dotnet` installs dotnet-cli to the default location --
- on windows, it's `$env:LOCALAPPDATA\Microsoft\dotnet`
- on osx and linux, it's `"$env:HOME/.dotnet"`

How about we remove the folder from the default location in `Install-Dotnet` before installing the new one?

> https://dotnet.github.io/getting-started/

I think we can skip this link, it's not very useful as we are using `2.0.0-preview`. And also, running `Install-Dotnet` will suffice. You don't need to run `Start-PSBootstrap`.